### DISCUSSION TITLE
aria2 rpc批量下载，多个弹幕能不能不打包，并直接推到aria2下载

### DISCUSSION THREAD

**Discussion by ashesofdream** [the1812/Bilibili-Evolved#2488] [功能建议 / Ideas]
使用nas的aria2下载，弹幕要单独放到nas上不太方便。

**Comment by the1812** [comment]
打包是在本地打的，不存在于 b 站服务器，而 RPC 需要发链接过去

**Reply by ashesofdream** [reply]
怎么本地打包呢？不是直接从bilibili下载到的xml文件吗？https://github.com/SocialSisterYi/bilibili-API-collect/blob/master/danmaku/danmaku_xml.md

**Reply by the1812** [reply]
你说 XML 啊, 那应该可以看下能不能直接发
另外两个格式是本地生成的就不行了

**Reply by ashesofdream** [reply]
xml 就行,市面上有支持播放的播放器

**Reply by ashesofdream** [reply]
啥时候这个才能提上日程啊？

**Reply by the1812** [reply]
没有日程, 近期在弄清爽首页. 等我想优化下载视频的时候会带上这个的
